src/core/config/transformer/improve_type_names.rs

```diff
@@ -86,7 +86,7 @@ impl<'a> CandidateGeneration<'a> {
                     .entry(field_info.type_of.to_owned())
                     .or_default();
 
-                let singularized_candidate = field_name.to_singular();
```

Comment:
Does the to_singular API need to be deleted?

Reply:
yes, removed now using `convert_case` for pascal case conversion

Reply:
It's part of the removed crate `Inflector`, and the functionality was replaced by the pluralize function in the following line.

https://docs.rs/Inflector/latest/inflector/string/singularize/fn.to_singular.html